lib/nerves_hub_link/application.ex

```diff
@@ -5,6 +5,8 @@ defmodule NervesHubLink.Application do
   alias NervesHubLink.Client
   alias NervesHubLink.Configurator
   alias NervesHubLink.Socket
+  alias NervesHubLink.Featuress
```

Comment:
```suggestion
  alias NervesHubLink.Features
```

Reply:
I think the naming is correct because Jon is referring to a lot of features. 🤣

Reply:
My bad. That should be `Featurez`